registry/session_v2.go

```diff
@@ -124,14 +124,15 @@ func (r *Session) HeadV2ImageBlob(ep *Endpoint, imageName, sumType, sum string,
 		return false, err
 	}
 	res.Body.Close() // close early, since we're not needing a body on this call .. yet?
-	switch res.StatusCode {
-	case 200:
+	switch {
```

Comment:
`switch` implies coverage over an input domain, which is what we want here. Arguably, default return should be pulled up into switch for clarity.